.github/linters/.eslintrc.yml

```diff
@@ -36,6 +36,12 @@ extends:
 
 rules:
   {
+    'import/no-unresolved': [
```

Comment:
We should not disable this.
Whatever the error we must fix it in another way.
If this is about the `parse-diff` module than we need to augment it in a types file (aglobal.d.ts ) ?

Reply:
we dont need to augment. parse-diff was coded in ts and exports types out of the box

Reply:
i reverted the changes here. it's for `@octokit/types` but we dont need it anymore